tools/bbc/bbc.cpp

```diff
@@ -196,10 +200,10 @@ static void convertFortranSourceToMLIR(
     // Continue to lower from MLIR down to LLVM IR. Emit LLVM and MLIR.
     pm.addPass(fir::createFIRToLLVMPass(nameUniquer));
     std::error_code ec;
-    llvm::ToolOutputFile outFile(outputFilename + ".ll", ec,
+    llvm::ToolOutputFile outFile(LLVMOutputFilename + ".ll", ec,
                                  llvm::sys::fs::OF_None);
     if (ec) {
-      llvm::errs() << "can't open output file " + outputFilename + ".ll";
+      llvm::errs() << "can't open output file " + LLVMOutputFilename + ".ll";
```

Comment:
Not sure I like this change. The `.mlir` and `.ll` file names will unexpectedly diverge unless both are set.

I think the test scripts should simply be programmed to use `-o -` to dump output to stdout. That actually follows the precedent of other LLVM tools.

Reply:
Also, having output from debug, like the `-print-ir-after-all` switch, mixed in with the "real" output isn't the most helpful default setting.

Reply:
Thanks, I was not a fan of this either, I had not thought about having the test doing `-o -`. Works great.